src/app/users/user-permission.ts

```diff
@@ -1,8 +1,6 @@
 import { User } from './user';
-import { UserPermissionScope } from './user-permission-scope';
+import { CommonPermissionResponse } from '../permissions/common-permission-response';
 
-export interface UserPermission {
-  owner: User;
-  scope: UserPermissionScope;
+export interface UserPermission extends CommonPermissionResponse {
```

Comment:
I suggest to add the suffix `Response` to `UserPermission` to have it conformed with the naming convention (`GlobalUserPermissionResponse`, `CommonPermissionResponse`).

Reply:
Thank you. Renamed